server/JsonStreamingServer/suppliers/JsonStreamingServer.Suppliers.FileStream/JsonStreamingServer.Suppliers.FileStream.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CsvHelper" Version="30.0.1" />
+    <PackageReference Include="Serilog" Version="2.12.0" />
```

Comment:
A supplier should not depend on Serilog if we want to introduce Serilog into the system we need to do it on the infrastructure level.

Reply:
Ok, removed